code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1630,3 +1630,14 @@
 	description = "blue sparkles that get everywhere"
 	color = "#4040FF" //A blueish color
 	glitter_type = /obj/effect/decal/cleanable/glitter/blue
+
+/datum/reagent/pukonium
```

Comment:
it's called bad_food on /tg/

https://github.com/tgstation/tgstation/blob/45d3d52af7a832e62bf83d614a3c971ee11d2cb1/code/modules/reagents/chemistry/reagents/toxin_reagents.dm#L481

```description = "The result of some abomination of cookery, food so bad it's toxic."```